sockeye/translate.py

```diff
@@ -67,40 +68,60 @@ def main():
                                                   *sockeye.inference.load_models(context,
                                                                                  args.max_input_len,
                                                                                  args.beam_size,
+                                                                                 args.batch_size,
                                                                                  args.models,
                                                                                  args.checkpoints,
                                                                                  args.softmax_temperature,
                                                                                  args.max_output_length_num_stds))
-        read_and_translate(translator, output_handler, args.input)
+
+        logger.info("Using batches of size %d", args.batch_size)
+        read_and_translate(translator, output_handler, args.portion_size, args.input)
 
 
 def read_and_translate(translator: sockeye.inference.Translator, output_handler: sockeye.output_handler.OutputHandler,
-                       source: Optional[str] = None) -> None:
+                       portion_size: int, source: Optional[str] = None) -> None:
```

Comment:
as mentioned above, I would prefer chunk_size

Reply:
Ok, but will conflate a bit with the other meaning of chunk inside inference.py - when splitting the over-sized sentences

Reply:
these could be refactored to sentence_chunk or something to make it explicit.